docs/design-reviews/decision-log/examples/memory/Trade-Studies/.template.md

```diff
@@ -0,0 +1,82 @@
+# Trade Study Template
```

Comment:
Do we need the template? I believe we already have a copy of the template in the playbook?

Reply:
Removed